src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -113,20 +112,60 @@ export const JiraConfigurationGet = async (
 				isGlobalInstall: installation.repository_selection === "all"
 			}));
 
+		/**
+		 * TODO: fetch the actual list of GHE servers
+		 *
+		 * The following array is a sample
+		 * [
+		 * 		{
+		 * 			id: 1,
+		 * 			url: "https:/github.internal.company.com",
+		 * 			applications: [
+		 * 				{
+		 * 					id: 1,
+		 * 					name: "ghe-app-for-jira",
+		 * 					successfulConnections: [1, 2, 3].map(() => successfulConnections[0])
+		 * 				},
+		 * 				{
+		 * 					id: 2,
+		 * 					name: "inter-team-app",
+		 * 					successfulConnections: [1, 2, 3, 4].map(() => successfulConnections[0])
+		 * 				}
+		 * 			]
+		 * 		},
+		 * 		{
+		 * 			id: 2,
+		 * 			url: "https:/github.internal.oooohanothercompany.com",
+		 * 			applications: [
+		 * 				{
+		 * 					id: 1,
+		 * 					name: "apps-galore",
+		 * 					successfulConnections: [1, 2, 3, 4, 5].map(() => successfulConnections[0])
+		 * 				},
+		 * 				{
+		 * 					id: 2,
+		 * 					name: "loose-canon",
+		 * 					successfulConnections: [1, 2, 3, 4].map(() => successfulConnections[0])
+		 * 				}
+		 * 			]
+		 * 		}
+		 * ]
+ 		 */
+		const gheServers = [];
+
 		const handleNavigationClassName = await booleanFlag(BooleanFlags.GHE_SERVER, false, jiraHost)
 			? "select-github-version-link"
 			: "add-organization-link";
 
 		res.render("jira-configuration.hbs", {
 			host: jiraHost,
-			successfulConnections,
-			failedConnections,
-			hasConnections: !!installations.total,
+			gheServers,
```

Comment:
We shouldn't be touching production code here. To be on the safer side, until the FF has been rolled out 100% in prof, I suggest that we create 2 jira-configuration views and pass data to each separately:

- the first would keep the current naming: jira-configuration.hbs - this would take in the data that currently exists in prod
- the second would be the new version: something like jira-configuration-new.hbs or jira-configuration-ghe.hbs - this would take in the data you are specifying in this PR.

Then set the page to be rendered based on whether the FF is on or not and pass the data

Reply:
The new view and css have been separated, which can be enabled by the feature flag `ghe_server`